src/web/app/pages-logs/logs-page.component.ts

```diff
@@ -56,6 +56,7 @@ export class LogsPageComponent implements OnInit {
   ];
   readonly EVENTS: LogEvent[] = [
     LogEvent.REQUEST_LOG, LogEvent.EXCEPTION_LOG, LogEvent.EMAIL_SENT, LogEvent.FEEDBACK_SESSION_AUDIT,
+    LogEvent.INSTANCE_LOG,
```

Comment:
Not really related to this PR but is there a reason why `LogEvent.DEFAULT_LOG` is not included here?

Reply:
`DEFAULT_LOG` is only a fallback for other JSON payloads that are not yet well-structured for whatever reason. In ideal situation no logs should ever be converted to that type.